src/std-types/word_counter.rs

```diff
@@ -0,0 +1,140 @@
+// ANCHOR: word_counter
+use std::collections::HashMap;
+use clap::Parser;
```

Comment:
We don't talk about `clap` in fundamentals. And, command-line interface is sort of out-of-scope for this exercise, anyway -- the assignment relates to counting words, and the segment is about types in `std`. This should probably just take a static string as input.